include/net/cloud.h

```diff
@@ -325,7 +326,8 @@ static inline int cloud_ping(const struct cloud_backend *const backend)
  * @return Time in milliseconds until next keep alive message is expected to
  *         be sent.
```

Comment:
As suggested earlier it would be good to add short info on what does the `UINT32_MAX` mean here.

Reply:
I'd actually prefer to return -1 for disabled keep-alive and document it like that in the cloud API. Translation from UINT32_MAX to -1 in the cloud backends from native API to cloud API could be changed to not just cast but specifically check for UINT32_MAX and then return -1 to make it clearer just from reading the code what happens.